### ISSUE TITLE
Magento2.1 documentation CategoryLinkRepositoryInterface

### ISSUE THREAD

**Issue by ghost** [magento/magento2#6371]
In Magento core file Magento\Catalog\Api\CategoryLinkRepositoryInterface has a typing mistake.
### Preconditions
1. Magento version 2.1
### Steps to reproduce
1. Go to Magento\Catalog\Api directory open CategoryLinkRepositoryInterface.php file
2. Before function `deleteByIds($categoryId, $sku);` is documentation where is not written what type $categoryId is, instead there is double  `* @param string $sku.`
### Expected result
1.  In file CategoryLinkRepositoryInterface.php has written variable $categoryId type - is it string, array ...  and has only once $sku variable type written.

**Comment by veloraven** [comment]
@Pistrik thank you for your report.
Please, format this issue according to the [Issue reporting guidelines](https://github.com/magento/magento2/wiki/Issue-reporting-guidelines): with steps to reproduce, actual result and expected result.

**Comment by veloraven** [comment]
According to [contributor guide](http://devdocs.magento.com/guides/v2.1/contributor-guide/contributing.html), tickets without response for two weeks should be closed.
If this issue still reproducible please feel free to create the new one: format new issue according to the [Issue reporting guidelines](https://github.com/magento/magento2/wiki/Issue-reporting-guidelines): with steps to reproduce, actual result and expected result and specify Magento version.
